### DISCUSSION TITLE
Jazzy Release Thread

### DISCUSSION THREAD

**Discussion by sea-bass** [moveit/moveit2#3163] [General]
This thread will cover Jazzy release planning.

**Comment by sea-bass** [comment]
Due to the merging of https://github.com/moveit/moveit2/pull/3143, which fixes a relatively critical bug, I think we're on track for another Jazzy release even if the latest one hasn't even made it.

Even though 2.12.0 is still queued up for the next Jazzy and Rolling syncs, what I'm thinking is to do a 2.12.1 specifically for Jazzy, whereas Rolling can just hang out until we decide to do a 2.13.0.

The sync will likely be end of December 2024, as the last 2 syncs were on October 18 and November 21.

Thoughts?

**Reply by mikeferguson** [reply]
Yeah, that seems like a pretty critical bugfix and worth another release (especially since the regression will be new to 2.12.0 AFAICT)

**Reply by sea-bass** [reply]
... and it's queued up!

https://github.com/ros/rosdistro/pull/43973

**Comment by sea-bass** [comment]
I think it's time for 2.12.2 to happen soon -- see https://github.com/orgs/moveit/discussions/3310#discussioncomment-12061924 for more details.

**Reply by sea-bass** [reply]
2.12.2 is now queued up for Jazzy: https://github.com/ros/rosdistro/pull/44488

**Comment by sea-bass** [comment]
Referring to https://github.com/moveit/moveit2/pull/3443#issuecomment-2805014485, we merged a nice to have fix that is ABI incompatible, meaning it's a good idea to get a new patch release out so the binaries and source are as synced up as possible.

2.12.3 is now queued up: https://github.com/ros/rosdistro/pull/45195
